docker/build_docker_oss4local.sh

```diff
@@ -33,6 +33,13 @@ else
     DEVICE_TYPE="$4"
 fi
 
+# process argument 5: support for parmetis
+if [ -z "$4" ]; then
+    USE_PARMETIS="false"
```

Comment:
We better to note the option here in the doc. We have changed the way to build docker image here.